client/.eslintrc.json

```diff
@@ -32,6 +26,7 @@
         "no-param-reassign": "off", // should refactor later
         "no-return-assign": "off", // should refactor later
         "spaced-comment": "off",
-        "arrow-body-style": "off" // should refactor later
+        "arrow-body-style": "off", // should refactor later
+        "linebreak-style": "off"
```

Comment:
https://github.com/Researchify/Researchify/pull/204#discussion_r703589285

Reply:
Link to Akshay's comment on linebreak-style. Maybe someone with experience can answer this?

Reply:
Hmm, cause if I don't turn it off, I'll get warnings in every file. But from limited research I did just now, the preferred way is to change the end of lines to `LF` to follow UNIX system. And we can change it by [changing the setting in vscode](https://qvault.io/clean-code/line-breaks-vs-code-lf-vs-crlf/). Anyways, I'll update this branch to change all the file to using `LF`

Reply:
See my comment I just posted: https://github.com/Researchify/Researchify/pull/204#discussion_r705197184
This should be ok.

Reply:
ok ill add it back